velox/exec/fuzzer/AggregationFuzzerBase.h

```diff
@@ -164,6 +164,7 @@ class AggregationFuzzerBase {
   static VectorFuzzer::Options getFuzzerOptions(
       VectorFuzzer::Options::TimestampPrecision timestampPrecision) {
     VectorFuzzer::Options opts;
+    VELOX_USER_CHECK_GE(FLAGS_batch_size, 0);
```

Comment:
Since this PR already fix the mod zero issue, do we still not allow batch_size to be 0?

There is a batch_size flag in RowNumberFuzzer too.

Reply:
Hi @pramodsatya, thank you for taking a look. Have you tried running the aggregation fuzzer, window fuzzer, join fuzzer, and row number fuzzer with --bathc_size=0? If they all work properly, I think we can avoid `VELOX_USER_CHECK_GE(FLAGS_batch_size, 0)` for all of them. If some of these fuzzers do not work with --batch_size=0, let's add the VELOX_USER_CHECK_GE and document what is needed for them to allow it.

Reply:
Thanks @kagamiori, I ran aggregate fuzzer and window fuzzer with `--batch_size=0` and don't see any errors. Join fuzzer and row number fuzzer expect the batch_size to be greater than 10, so removed the check `VELOX_USER_CHECK_GE(FLAGS_batch_size, 0)` for all of them.